### DISCUSSION TITLE
Directions of max intensity pole figures

### DISCUSSION THREAD

**Discussion by maxbut38** [mtex-toolbox/mtex#1969] [Ask Anything]
Hi,

I am looking for a method allowing to obtain the direction (meaning hkl values) of the maximum value of a pole figure. 
I plot the pole figures using plotPDF function but i did not manage to get the maximum position and transform the (X,Y) value to the hkl direction .
```matlab
ebsd = EBSD.load(fname,CS,'interface','ang','convertEuler2SpatialReferenceFrame');
plotPDF(ebsd('Al').orientations,Miller(0,1,0,ebsd('Al').CS),'antipodal','smooth','grid');
``` 
Bests,
Maxence

**Comment by zmichels** [comment]
I think you want to first compute an ODF from the orientations using calcDensity(). Then you can use max() on the ODF:

[value,ori] = max(odf)

**Comment by ralfHielscher** [comment]
Hi Maxence,

in case you are interested in the maximum of the pole figure you can do
```matlab
odf = calcDensity(ebsd('Al').orientations)
pdf = calcPDF(odf,Miller(0,1,0,ebsd('Al').CS),'antipodal')
[value,pos] = max(pdf)
```
However this gives you a direction `pos`. However due to the nature of a pole figure it is a direction with respect to specimen coordinates - no (hkl) :).

Ralf.
